static/webmention.js

```diff
@@ -152,7 +152,7 @@ A more detailed example:
         var html = '<a class="reaction" rel="nofollow ugc" title="' + who + ' ' +
             response + '" href="' + r[mentionSource] + '">';
         if (r.author && r.author.photo) {
-            html += '<img src="' + entities(r.author.photo) + '" loading="lazy">';
+            html += '<img src="' + entities(r.author.photo) + '" loading="lazy" decoding="async" alt="">';
```

Comment:
Finally got around to looking at this at a computer instead of on my cellphone. :) I'd just set the `alt` attribute to `who`.

Reply:
Do you know **_exactly_** how to do that? I tried `alt="' + who + '"` but it resulted in a JS error.
If you let me know I'll push a new commit to this PR. (JavaScript isn't really my forte :-) )
Otherwise, feel free to merge this as it is and/or make an additional commit/PR with the correct alt text to this PR.

Reply:
What JS error did you get? Because that's precisely how to do it... weird. 

But sure I can just accept this as-is and do the rest myself.